File: codewars/6kyu/c/3_5_fizzbuzz.c
int solution(int number) {
  int sum=0;
  int i=2;
  number--;
	while(i++<number){
    if (i % 3 == 0 || (i>=5 && i % 5 == 0)){
      sum+=i;
    }
  }
  return sum;
}

File: codewars/6kyu/py/3_5_fizzbuzz.py
def solution(number):
    sum=0
    for i in range(3,number):
        if i >= 15 and i % 15 == 0:
            print(i)
            sum+=i
        elif i >= 5 and i % 5 == 0:
            print(i)
            sum+=i
        elif i % 3 == 0:
            print(i)
            sum+=i
    
    return sum
